api/build.gradle

```diff
@@ -0,0 +1,7 @@
+subprojects {
```

Comment:
Whoa, this is an annoying bug. And an old one 😃

How about we rename the directories instead?